### DISCUSSION TITLE
Enhanced Multi-Monitor Support

### DISCUSSION THREAD

**Discussion by mismatcharm** [elementary/triage#341] [Ideas]
I use two monitors. Sometimes it's harder to work with one of them than the other one, as one is the "primary" display, and therefore has more features. I would like to see the monitor that is not the primary display get more features. The multitasking view application built-in to eOS doesn't let me add additional workspaces for my second monitor, and I'd love it if it would let me add additional workspaces. Without it, I end up dragging apps I'm not actively using to my primary monitor just to store it there until later, and this isn't optimal for me. There are other features I'd love to have on my secondary display, such as the panel as the dock, but I can accept that as a stylistic decision (though, it does get annoying in certain situations to have different resolutions for similar windows on different monitors due to the panel).

**Comment by iniquitor** [comment]
I have a laptop with a 4K resolution and a 34in QHD monitor. I can only have one scaling set for both and that is very frustrating to just have one. It makes my laptop monitor quite useless if I stick with my QHD monitor as the primary and vice versa.

Some enhancements for multiple monitors I would love to see are:
- Have the dock visible on both monitors, displaying apps running on each
- Support for individual scaling settings on each monitor
- Option to move windows to other monitors and back

**Comment by edmundlaugasson** [comment]
Would propose also to set default shortcut key Super+P to quickly switch between connected monitors and decide, whether to use:
- Only Display 1
- Extend desktop to the right, left, upwards, downwards
- Mirror displays
- Only Display 2

E.g. [XFCE has such feature](https://wiki.xfce.org/design/xfce4-settings/display#status_quo).

**Reply by lenemter** [reply]
I think this is already exists.
